client-src/elements/chromedash-feature-detail.ts

```diff
@@ -71,37 +75,33 @@ export const DETAILS_STYLES = [
 const LONG_TEXT = 60;
 
 class ChromedashFeatureDetail extends LitElement {
-  static get properties() {
-    return {
-      appTitle: {type: String},
-      featureLinks: {type: Array},
-      user: {type: Object},
-      canEdit: {type: Boolean},
-      feature: {type: Object},
-      gates: {type: Array},
-      process: {type: Object},
-      progress: {type: Object},
-      anyCollapsed: {type: Boolean},
-      selectedGateId: {type: Number},
-      openStage: {type: Number},
-    };
-  }
-
-  constructor() {
-    super();
-    this.appTitle = '';
-    this.user = {};
-    this.canEdit = false;
-    this.feature = {};
-    this.gates = [];
-    this.process = {};
-    this.progress = {};
-    this.anyCollapsed = true;
-    this.previousStageTypeRendered = 0;
-    this.sameTypeRendered = 0;
-    this.selectedGateId = 0;
-    this.openStage = 0;
-  }
+  @property({type: String})
+  appTitle = '';
+  @property({type: Array, attribute: false})
```

Comment:
As far as I can tell, the `type` field is only used when an attribute is defined (to convert string attribute values into typed property values), so we should generally have either `attribute:false` or `type: Something` but not both.